rq/queue.py

```diff
@@ -699,10 +715,13 @@ def lpop(cls, queue_keys, timeout, connection: t.Optional['Redis'] = None):
         if timeout is not None:  # blocking variant
             if timeout == 0:
                 raise ValueError('RQ does not support indefinite timeouts. Please pick a timeout value > 0')
+            logger.debug(f"Starting blocking pop operation for queues {green(queue_keys)} with timeout of {timeout}")
```

Comment:
Nitpick: we should make the blocking pop terminology consistent. `Starting BLPOP for queues {green(queue_keys)} with timeout of {timeout}`